src/main/java/uk/gov/pay/api/model/links/PaymentWithAllLinks.java

```diff
@@ -42,7 +42,7 @@ public PaymentWithAllLinks(String chargeId, long amount, PaymentState state, Str
                                String reference, String email, String paymentProvider, String createdDate, SupportedLanguage language,
                                boolean delayedCapture, RefundSummary refundSummary, SettlementSummary settlementSummary, CardDetails cardDetails,
                                List<PaymentConnectorResponseLink> paymentConnectorResponseLinks, URI selfLink, URI paymentEventsUri, URI paymentCancelUri,
-                               URI paymentRefundsUri, Long corporateCardSurcharge, Long totalAmount) {
+                               URI paymentRefundsUri, URI captureUri, Long corporateCardSurcharge, Long totalAmount) { // Add capture URL!
```

Comment:
Please remove the `// Add capture URL!` comment (that’s kind of my fault because I typed it on your machine!)